client/src/state/redux/tables/operations.js

```diff
@@ -125,7 +125,7 @@ const transactionListSearch = (channel, query) => dispatch =>
 
 /* istanbul ignore next */
 const transactionList = channel => dispatch =>
-	get(`/api/txList/${channel}/0/0/`)
+	get(`/api/txList/${channel}/0/0/?page=1&size=10`)
```

Comment:
it would be good if `page` and `size` are pass from the components.

Reply:
resolved the review comments